clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala

```diff
@@ -187,7 +187,7 @@ class MoveExecutor[TI <: TransferIndex](moveCommand: MoveCommand[TI])
             if (moves.forall(_.isRight)) {
               logger.info("All objects were successfully deleted in gcloud.")
             } else {
-              val notDeleted = moves.filter(_.isLeft)
+              val notDeleted = moves.filter(_.isLeft).map(_.left.get)
```

Comment:
Codacy will probably get mad about the `get`. This should appease it:
`moves.flatMap(_.left.toOption)`